src/sentry/static/sentry/app/views/stream/stream.jsx

```diff
@@ -637,6 +638,14 @@ const Stream = createReactClass({
   },
 
   renderGroupNodes(ids, statsPeriod) {
+    // Restrict this guide to only show on a project with less than two issues
+    // because anchors light up all issues on the page. Also restrict to new users (joined<30 days)
+    let userDateJoined = new Date(ConfigStore.get('user').dateJoined);
+    let dateCutoff = new Date();
+    dateCutoff.setDate(dateCutoff.getDate() - 30);
+
+    let hasGuideAnchor = ids.length <= 2 && userDateJoined > dateCutoff;
```

Comment:
Hmm will this get triggered on searches etc. that return <= 2 results? If so, can we add another check for "is user's default view"?

Reply:
It gets triggered on searches as well. What's the concern here?